### ISSUE TITLE
Unable to POST to upload binary file

### ISSUE THREAD

**Issue by adrian-a-graham** [IamKingWaiMark/HTTPRequestForBlueprintsSupport#8]
I'm trying to use the plugin to upload a binary file (audio recorded in UE) to a remote server using HTTP POST.
I am able to successfully upload a test file using the command line:

```
curl -X POST https://example.com/upload_audio \
  -F "test=one" \
  -F "fileUpload=@tmpMicInput.wav" \
  -H "Content-Type: multipart/form-data"

```
However, I'm unable to perform the same with the following Blueprint structure:

<img width="1186" alt="Screenshot 2024-04-24 at 6 33 55 PM" src="https://github.com/IamKingWaiMark/HTTPRequestForBlueprintsSupport/assets/69984844/4e6ff6b3-e096-42bc-a1a5-f098df51ce75">

The Python code handling this on the server end is saying the expected payload is empty:
```
request_payload: ImmutableMultiDict([])
```

Am I doing something wrong in my BP? Is POST not supported with the plugin? I'm using v5.3, by the way.

**Comment by IamKingWaiMark** [comment]
Hello, does @tmpMicInput.wav point to a file?

**Comment by adrian-a-graham** [comment]
Correct, that file lives in the dir `$PROJECT/Saved/BouncedWavFiles`, which is the default write location for when you record audio from the microphone. No matter what the path I put in there, I still get an empty payload.
I did try changing the path to `@/Saved/BouncedWavFiles/tmpMicInput.wav` and got the same error.

**Comment by IamKingWaiMark** [comment]
Not sure if it is possible with the Params as that will be part of the request url.

For example, that would be https://example.com/upload_audio?fileUpload=@tmpMicInput

For post requests, it should be using the body pin. Have not tried using a file.

**Comment by adrian-a-graham** [comment]
I’ll try changing to body content. What format should the body text be in?
